### ISSUE TITLE
Increase spacing between values

### ISSUE THREAD

**Issue by Castle33** [ShawnLin013/NumberPicker#101]
I haven't been able to increase the padding between values without having to increase the widget height (for vertical picker). Is this functionality available or there is no way to do it?

The reason for this is that I cannot give the widget more height (limited space) but I still need the text size to be big enough, so  the values of each line end up overlapping. Here is a visual example of the problem:

![image](https://user-images.githubusercontent.com/17215270/38358946-1aa03fd0-38c7-11e8-8845-932da7811d9a.png)

**Comment by AccuPhoenix01** [comment]
Yes I also want to be able to adjust spacing (larger AND smaller) between items.

**Comment by vw2k3** [comment]
@ShawnLin013 Please help to solve this issue🙏

**Comment by dbarr33** [comment]
@ShawnLin013 I also have the need to add spacing between items. I have a branch with a working solution. Is it possible for me to get access to create prs?
![Screenshot_1606915873](https://user-images.githubusercontent.com/17283517/100880005-31bc2380-347a-11eb-9a33-773c9b1ef140.png)

**Comment by ShawnLin013** [comment]
> @ShawnLin013 I also have the need to add spacing between items. I have a branch with a working solution. Is it possible for me to get access to create prs?

@dbarr33 
Sure, you need to fork this repository and make your changes first, then create a pull request from the forked repository without any permissions.

**Comment by dbarr33** [comment]
@Castle33 this pr has a fix for this issue. https://github.com/ShawnLin013/NumberPicker/pull/185

@ShawnLin013 any idea when this fix will get released? Thanks for the quick review.
